### PR TITLE
add restart always to the webservice

### PR DESCRIPTION
**Pull request by phulstaert**
I often have only my db running. Went to check and saw that only the db has a restart setting. Added it to the webservice.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -17,6 +17,7 @@ services:
   leantime:
     image: leantime/leantime:latest
     container_name: leantime
+    restart: always
```
Comment (funkywaddle) [inline docker-compose.yml:20]: I prefer ```restart: unless-stopped```

Comment (phulstaert) [reply docker-compose.yml:20]: Yes, this would be better, but we have to be consistent between web and db.

Comment (funkywaddle) [reply docker-compose.yml:20]: then change the db one too

**Comment by marcelfolaron** [comment]
I assume this is obsolete now.

**Pull request closed without merging**